examples/dev-mashup/connect.js

```diff
@@ -0,0 +1,53 @@
+import enigma from 'enigma.js';
+
+export default function connect() {
+  const loadSchema = () => fetch('https://unpkg.com/enigma.js/schemas/3.2.json').then((response) => response.json());
```

Comment:
consider using a newer schema version 3.2 is almost 4 years old (12.612.0.json looks to be the latest)